src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -0,0 +1,411 @@
+            //     _stateDb.Commit();
+            //     _codeDb.Commit();
+            // }
+        }
+        
+        private void Execute(ContractEntry entry, Delta delta, ITxTracer txTracer, bool readOnly, Address recipientOverride = null)
+        {
+            var stateUpdate = ToStateUpdate(delta);
+            long gasLimit = entry.GasLimit;
+            UInt256 gasPrice = entry.GasPrice;
+            Address recipient = recipientOverride ?? GetAccountAddress(entry.Base.ReceiverPublicKey);
+            Address sender = GetAccountAddress(entry.Base.SenderPublicKey);
+            var value = entry.Amount.ToUInt256();
+
+            IReleaseSpec spec = _specProvider.GetSpec(stateUpdate.Number);
+            byte[] machineCode = entry.IsValidDeploymentEntry ? entry.Data.ToByteArray() : null;
+            byte[] data = entry.Data.ToByteArray();
+
+            // if (_logger.IsTrace) _logger.Trace($"Executing tx {transaction.Hash}");
+            if (sender == null)
+            {
+                // TraceLogInvalidTx(entry, "SENDER_NOT_SPECIFIED");
+                QuickFail(entry, stateUpdate, txTracer, readOnly);
+                return;
+            }
+            
+            long intrinsicGas = CalculateIntrinsicGas(entry, spec);
+            
+            //if (_logger.IsTrace) _logger.Trace($"Intrinsic gas calculated for {entry}: " + intrinsicGas);
+
+            if (gasLimit < intrinsicGas)
+            {
+                // TraceLogInvalidTx(transaction, $"GAS_LIMIT_BELOW_INTRINSIC_GAS {gasLimit} < {intrinsicGas}");
+                QuickFail(entry, stateUpdate, txTracer, readOnly);
+                return;
+            }
+
+            if (gasLimit > stateUpdate.GasLimit - stateUpdate.GasUsed)
+            {
+                // TraceLogInvalidTx(transaction,
+                // $"BLOCK_GAS_LIMIT_EXCEEDED {gasLimit} > {stateUpdate.GasLimit} - {stateUpdate.GasUsed}");
+                QuickFail(entry, stateUpdate, txTracer, readOnly);
+                return;
+            }
+
+            if (!_stateProvider.AccountExists(sender))
+            {
+                // TraceLogInvalidTx(transaction, $"SENDER_ACCOUNT_DOES_NOT_EXIST {sender}");
+                if (gasPrice == UInt256.Zero)
+                {
+                    _stateProvider.CreateAccount(sender, UInt256.Zero);
+                }
+            }
+
+            UInt256 senderBalance = _stateProvider.GetBalance(sender);
+            if ((ulong) intrinsicGas * gasPrice + value > senderBalance)
+            {
+                // TraceLogInvalidTx(transaction, $"INSUFFICIENT_SENDER_BALANCE: ({sender})_BALANCE = {senderBalance}");
+                QuickFail(entry, stateUpdate, txTracer, readOnly);
+                return;
+            }
+
+            if (entry.Base.Nonce != _stateProvider.GetNonce(sender))
+            {
+                // TraceLogInvalidTx(transaction, $"WRONG_TRANSACTION_NONCE: {transaction.Nonce} (expected {_stateProvider.GetNonce(sender)})");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4350015)